actions1/action.yml

```diff
@@ -0,0 +1,5 @@
+name: 'CLA Check'
```

Comment:
Why can this be added to the actions folder that we have now? Or if it is a problem you can always add folders into the actions folder that we have now.

Reply:
Added this to a separate folder within actions since it is a different GitHub action.